documentation/site/content/samples/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -112,418 +112,226 @@ weblogic-operator-webhook-868db5875b-55v7r   1/1     Running   0          86s
+  - The WebLogic credentials secret is required and must contain `username` and `password` fields. You reference it in `spec.webLogicCredentialsSecret` field of Domain YAML file and macros in the `domainInfo.AdminUserName` and `domainInfo.AdminPassWord` fields your model YAML file.
+  - Delete a secret before creating it, otherwise the create command will fail if the secret already exists..
+  - Name and label the secrets using their associated domain UID to clarify which secrets belong to which domains and make it easier to clean up a domain.
 
 ##### Create WebLogic Domain
-You will use the `kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain.sh` script to create the WLS domain on the persistent volume you created previously.
-
-{{% notice note %}} The `create-domain.sh` script and its inputs file are for demonstration purposes _only_; its contents and the domain resource file that it generates for you might change without notice. In production, we strongly recommend that you use the WebLogic Image Tool and WebLogic Deploy Tooling (when applicable), and directly work with domain resource files instead.
-{{% /notice%}}
-
-You need to set up the domain configuration for the WebLogic domain.
-
-1. Check if resources are ready.
-
-   If you used the automation script to create the AKS cluster, skip this step and go to step 2.
-
-   If you created Azure resources, step-by-step, according to the previous steps, then validate that all the resources above were created by using the script `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/validate.sh`.
-
-   Use the following commands to check if the resources are ready:
+Now, you deploy a `sample-domain1` domain resource and an associated `sample-domain1-cluster-1` cluster resource using a single YAML resource file which defines both resources. The domain resource and cluster resource tells the operator how to deploy a WebLogic domain. They do not replace the traditional WebLogic configuration files, but instead cooperate with those files to describe the Kubernetes artifacts of the corresponding domain.
 
-   ```
-   # cd kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service
-   ```
-   ```shell
-   $ ./validate.sh -g ${AKS_PERS_RESOURCE_GROUP} \
-    --aks-name ${AKS_CLUSTER_NAME} \
-    --file-share ${AKS_PERS_SHARE_NAME} \
-    --storage-account ${AKS_PERS_STORAGE_ACCOUNT_NAME} \
-    --domain-uid domain1 \
-    --pvc-name ${NAME_PREFIX}-azurefile-${TIMESTAMP} \
-    --secret-docker ${SECRET_NAME_DOCKER}
-   ```
+Copy the contents of the [WLS domain resource YAML file](https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/{{< latestMinorVersion >}}/kubernetes/samples/scripts/create-weblogic-domain/domain-on-pv/domain-resources/WLS/domain-on-pv-WLS-v1.yaml) file that is included in the sample source to a file called `~/azure/weblogic-on-aks/domain-resource.yaml` or similar.
```

Comment:
Please note that the WLS domain resource YAML file uses a Persistent volume of type `hostPath`. This only works on the single node cluster. For AKS, you'll need to modify the PV/PVC section to create a StorageClass and use that in PVC configuration. Alternatively, you can use PV of type `NFS` but using `StorageClass` is recommended. See the latest `Domain resources` section in the sample documentation at -> https://oracle.github.io/weblogic-kubernetes-operator/samples/domains/domain-home-on-pv/sample/#domain-resource

Reply:
Thank you @ankedia, from the context of the existing document, we will use `NFS` instead of `hostPath`.

Reply:
Sure. The existing (3.4 or 4.0) AKS Domain on Pv sample document uses the Azure Files Container Storage Interface (CSI) drivers to manage the NFS file share, with provisioner `file.csi.azure.com` and then creates the below StorageClass. Is that what you are implying by `NFS`?

```
cat <<EOF
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: azurefile-csi-nfs
provisioner: file.csi.azure.com
allowVolumeExpansion: true
parameters:
  protocol: nfs
mountOptions:
  - nconnect=4
EOF
```

Reply:
Yes, that's right. In this sample, we will use this instead of `hostPath`.

Reply:
I just noticed that this PR was still here. Sorry about that! @ankedia, are your concerns resolved?

Reply:
@rjeberhard Yes.